### DISCUSSION TITLE
MediaElement not showing on screen controls (MacCatalyst + iOS)

### DISCUSSION THREAD

**Discussion by StephanHartmann** [CommunityToolkit/Maui#1100] [Q&A]
Hello,

I embedded a MediaElement in my app and it is not showing the on screens controls (I enabled it in XAML).

So I build a sample project where I added the MediaElementPage and some other required sources from the toolkit sample app.
My first result was that the on screen controls were also missing.
Then I added more and more source from the toolkit sample app until I got it working.

I ended with the original navigation with the ViewsGalleryPage to the MediaElementPage.
Then I added the same MediaElementPage a second time as a regular page, and the on screen controls don't show in this case.

I'm not so deep in Maui that I can say if it is my fault or if it is bug in media element.

I have set up a repository with the source and a description with images and a short video.

I hope that someone can help me.

The video is also attached here:

https://user-images.githubusercontent.com/10873640/226037716-df467ead-80e3-4e76-895d-f365492dffdc.mp4

**Comment by PeterVanVogelpoel** [comment]
Same issue here.

**Comment by pictos** [comment]
@StephanHartmann can you please share the link of your sample?

**Reply by PeterVanVogelpoel** [reply]
https://github.com/StephanHartmann/maui-mediaelement-test

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
